test/libsolidity/syntaxTests/fallback/payable_fallback_with_inherited_receive.sol

```diff
@@ -0,0 +1,7 @@
+contract A {
+    receive() external payable { }
+}
+contract C is A {
+    fallback() external payable { }
+}
+// TODO: should there be a warning here?
```

Comment:
@ekpyron was anything decided regarding this?

Reply:
I think there was a tendency not to warn, but I'm not sure.

Reply:
But the `receive` function is inherited, right?

Reply:
Yes - so ``C`` has both fallback and receive and we don't warn.

Reply:
Ok!